src/main/scala/com/wavesplatform/state2/StateStorage.scala

```diff
@@ -17,9 +17,14 @@ class StateStorage private(file: Option[File]) extends VariablesStorage(createMV
 
   override protected val Version = 2
 
+  @volatile private[state2] var lastUpdated: Long = 0L
```

Comment:
Please clean this up, make a public getter and private setter, package-private vars is a mess